playbooks/roles/deploy-osh/templates/horizon.yaml.j2

```diff
@@ -1,14 +1,15 @@
 ---
 {% if developer_mode %}
```

Comment:
you can remove if developer_mode nowadays, and remove the upstream patch from the patch list.

Reply:
yay!